packages/contracts/migrations/development/Deploy001_Diamond_Dollar.s.sol

```diff
@@ -0,0 +1,308 @@
+                functionSelectors: selectorsOfAccessControlFacet
+            })
+        );
+        cuts[1] = (
+            FacetCut({
+                facetAddress: address(diamondCutFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfDiamondCutFacet
+            })
+        );
+        cuts[2] = (
+            FacetCut({
+                facetAddress: address(diamondLoupeFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfDiamondLoupeFacet
+            })
+        );
+        cuts[3] = (
+            FacetCut({
+                facetAddress: address(managerFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfManagerFacet
+            })
+        );
+        cuts[4] = (
+            FacetCut({
+                facetAddress: address(ownershipFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfOwnershipFacet
+            })
+        );
+        cuts[5] = (
+            FacetCut({
+                facetAddress: address(twapOracleDollar3PoolFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfTWAPOracleDollar3poolFacet
+            })
+        );
+        cuts[6] = (
+            FacetCut({
+                facetAddress: address(ubiquityPoolFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfUbiquityPoolFacet
+            })
+        );
+
+        // deploy diamond
+        diamond = new Diamond(diamondArgs, cuts);
+
+        // stop sending owner transactions
+        vm.stopBroadcast();
+
+        //=======================
+        // Diamond permissions
+        //=======================
+
+        // start sending admin transactions
+        vm.startBroadcast(adminPrivateKey);
+
+        AccessControlFacet accessControlFacet = AccessControlFacet(
+            address(diamond)
+        );
+
+        // grant diamond dollar minting and burning rights
+        accessControlFacet.grantRole(
+            DOLLAR_TOKEN_MINTER_ROLE,
+            address(diamond)
+        );
+        accessControlFacet.grantRole(
+            DOLLAR_TOKEN_BURNER_ROLE,
+            address(diamond)
+        );
+
+        // stop sending admin transactions
+        vm.stopBroadcast();
+
+        //=========================
+        // UbiquiPoolFacet setup
+        //=========================
+
+        // start sending admin transactions
+        vm.startBroadcast(adminPrivateKey);
+
+        UbiquityPoolFacet ubiquityPoolFacet = UbiquityPoolFacet(
+            address(diamond)
+        );
+
+        // add collateral LUSD token
+        uint256 poolCeiling = 10_000e18; // max 10_000 of collateral tokens is allowed
+        address lusdAddress = 0x5f98805A4E8be255a32880FDeC7F6728C6568bA0;
```

Comment:
i think we should look for a visibility to this state vars, usually addresses are set `public`

Reply:
This is the migration script contract which we will never call directly so it doesn't seem to make sense to make some state variables public unless I'm missing smth